packages/web/src/components/gcds-link/stories/gcds-link.stories.js

```diff
@@ -0,0 +1,216 @@
+export default {
+  title: 'Components/Link',
+
+  parameters: {
+    actions: {
+      argTypesRegex: '^on.*',
+      handles: ['click', 'change', 'focus', 'blur'],
+    },
+  },
+
+  argTypes: {
+    // Props
+    size: {
+      control: { type: 'radio' },
+      options: ['regular', 'small'],
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'regular' },
+      },
+    },
+    display: {
+      control: 'select',
+      options: ['inline', 'inline-block', 'block'],
```

Comment:
Can you set `inline` as the default value, please?

Reply:
Reminder for myself: set the events & properties tab to use the default values